Firestore/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Unreleased
-- [changed] If the SDK's attempt to connect to the Cloud Firestore backend
+
+# v0.10.4
+- [changed] If the SDKs attempt to connect to the Cloud Firestore backend
```

Comment:
This one really is possessive, not plural, I swear (feel free to rephrase if you don't think it reads right though).